packages/cli-node/src/monorepo/PackageGraph.ts

```diff
@@ -50,7 +50,7 @@ export interface BackstagePackageJson {
     /**
      * The ID of the plugin if this is a plugin package. Must always be set for plugin and module packages, and may be set for library packages.
      */
-    pluginId?: string;
+    pluginId?: string | null;
```

Comment:
maybe mention in the comment above what null means?

Reply:
Yep added a bit about that too